tests/platform_tests/test_intf_fec.py

```diff
@@ -20,6 +21,16 @@
     "100G", "200G", "400G", "800G", "1600G"
 ]
 
+FEC_CONFIG_DEFAULT_RETRIES = 5
+FEC_CONFIG_NON_COPPER_RETRIES = 10
+
+
+@pytest.fixture(scope="module", autouse=True)
+def disable_bgp(duthost):
```

Comment:
@nhe-NV where is the fixture used? why is this fixture in this file related to fec

Reply:
Hi @prgeor  the fixture is autouse.
Why we need it: 
When we choose the ports for the test, if on the ports there lot of routes on it, it causes results in oper up notification processed very late as orchagent has to reprogram all the 12 k routes with new nexthop information after a link is removed. To avoid such case, we disable the bgp before the test, and enable it back. to make the test much more stable.